opentelemetry-sdk/src/metrics/internal/sum.rs

```diff
@@ -26,10 +29,24 @@ impl<T: Number<T>> ValueMap<T> {
 impl<T: Number<T>> Aggregator<T> for ValueMap<T> {
     fn aggregate(&self, measurement: T, attrs: AttributeSet) {
         if let Ok(mut values) = self.values.lock() {
-            values
-                .entry(attrs)
-                .and_modify(|val| *val += measurement)
-                .or_insert(measurement);
+            let size = values.len();
+            match values.entry(attrs) {
+                Entry::Occupied(mut occupied_entry) => {
+                    let sum = occupied_entry.get_mut();
+                    *sum += measurement;
+                }
+                Entry::Vacant(vacant_entry) => {
+                    if self.check_stream_cardinality(size) {
+                        vacant_entry.insert(measurement);
+                    } else {
+                        values
+                            .entry(STREAM_OVERFLOW_ATTRIBUTE_SET.clone())
+                            .and_modify(|val| *val += measurement)
+                            .or_insert(measurement);
+                        println!("Warning: Maximum data points for metric stream exceeded. Entry added to overflow.");
```

Comment:
feedback from SIG meeting: 
Use standard error handling mechanism, instead of println.

Reply:
use the std mechanism now, @TommyCpp can explore how to enable it for Warning level internal logs.

Reply:
Done